src/main/java/org/jabref/gui/groups/GroupDialogView.java

```diff
@@ -188,4 +231,87 @@ private void texGroupBrowse() {
     private void openHelp() {
         viewModel.openHelpPage();
     }
+
+    @FXML
+    private void openIconPicker() {
+        ObservableList<Ikon> ikonList = FXCollections.observableArrayList();
+        FilteredList<Ikon> filteredList = new FilteredList<>(ikonList);
+
+        ikonList.addAll(allOf(MaterialDesignA.class));
```

Comment:
```suggestion
         Set<IkonProvider<Ikon>> result = ServiceLoader.load(IkonProvider.class.getModule().getLayer(), IkonProvider.class).stream()
                                                         .filter(provider -> provider.get().getClass() != JabrefIconProvider.class)
                                                         .map(Provider::get).collect(Collectors.toSet());

        for (IkonProvider provider: result) {
            ikonList.addAll(allOf(provider.getIkon()));
        }
```

Reply:
Something seems to be odd. Works with eclipse, but not with gradlew / javac

```error: incompatible types: cannot infer type-variable(s) R,A,CAP#1,T#2
                                                      .map(ServiceLoader.Provider::get).collect(Collectors.toSet());
                                                                                               ^
    (argument mismatch; Collector<IkonProvider<Ikon>,CAP#2,Set<IkonProvider<Ikon>>> cannot be converted to Collector<? super IkonProvider,CAP#2,Set<IkonProvider<Ikon>>>)
  where R,A,T#1,T#2 are type-variables:
    R extends Object declared in method <R,A>collect(Collector<? super T#1,A,R>)
    A extends Object declared in method <R,A>collect(Collector<? super T#1,A,R>)
    T#1 extends Object declared in interface Stream
    T#2 extends Object declared in method <T#2>toSet()
  where CAP#1,CAP#2 are fresh type-variables:
    CAP#1 extends Object from capture of ?
    CAP#2 extends Object from capture of ?```